test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -145,3 +172,178 @@ def test_t6(self):
+        resc_flag = ["nh", "nh"]
+
+        self.create_mult_mom_resources(resc_name, resc_type,
+                                       resc_flag, script_body)
+
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Submit a job that requests mom dynamic resource
+        attr = {"Resource_List." + resc_name[0]: 2,
+                "Resource_List." + resc_name[1]: 3}
+        j = Job(TEST_USER, attrs=attr)
+        jid = self.server.submit(j)
+
+        # The job should run
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+        # Submit a job that requests more value
+        # than available for bolth mom resources.
+        attr = {"Resource_List." + resc_name[0]: 4,
+                "Resource_List." + resc_name[1]: 4}
+        j = Job(TEST_USER, attrs=attr)
+        jid = self.server.submit(j)
+
+        # The job shouldn't run
+        c = "Can Never Run:"
+        c += " Insufficient amount of resource: foo (R: 4 A: 3 T: 3)"
+        self.server.expect(JOB, {'job_state': 'Q', 'comment': c},
+                           id=jid, attrop=PTL_AND)
+
+    def test_t10(self):
+        """
+        multiple host level resource with one or more incorrect values.
+        Submit different jobs with different resource types
+        """
+
+        resc_name = ["foo", "foo1"]
+        resc_type = ["float", "size"]
+        script_body = ["/bin/echo 3", "/bin/echo -2"]
+        resc_flag = ["nh", "nh"]
+
+        self.create_mult_mom_resources(resc_name, resc_type,
+                                       resc_flag, script_body)
+
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Submit a job that requests mom dynamic resource
+        attr = {"Resource_List." + resc_name[0]: 2,
+                "Resource_List." + resc_name[1]: 1}
+        j = Job(TEST_USER, attrs=attr)
+        jid = self.server.submit(j)
+
+        # The job shouldn't run
+        c = "Can Never Run:"
+        c += " Insufficient amount of resource: foo1 (R: 1kb A: 0kb T: 0kb)"
+        self.server.expect(JOB, {'job_state': 'Q', 'comment': c},
+                           id=jid, attrop=PTL_AND)
+
+    def test_t11(self):
+        """
+        delete a resource while jobs are running/queued.
+        """
+        resc_name = "foo"
+        self.create_mom_resources(resc_name, "string", "h", "/bin/echo abc")
+
+        a = {'scheduling': 'True'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        # Submit a job that requests mom dynamic resource
+        attr = {"Resource_List." + resc_name: 'abc'}
+        j = Job(TEST_USER, attrs=attr)
+        jid = self.server.submit(j)
+
+        # The job should run
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+
+        try:
+            self.server.manager(MGR_CMD_DELETE, RSC, id='foo')
+        except PbsManagerError as e:
+            self.logger.info(e.message)
```

Comment:
This will be confusing.  You are logging an error message, but passing your test.  Either just 'pass' here or log "Correctly unable to delete busy resource"